parameter_files/fates_params_14pfts.cdl

```diff
@@ -203,6 +207,51 @@ variables:
 	float fates_branch_turnover(fates_pft) ;
 		fates_branch_turnover:units = "yr-1" ;
 		fates_branch_turnover:long_name = "turnover time of branches" ;
+
+	float fates_prt_unit_gr_resp(fates_prt_organs,fates_pft) ;
+		fates_prt_unit_gr_resp:units = "gC/gC" ;
+		fates_prt_unit_gr_resp:long_name = "Unit growth respiration rate per organ" ;
+
+	float fates_prt_nitr_stoich_p1(fates_prt_organs,fates_pft) ;
+		fates_prt_nitr_stoich_p1:units = "(gN/gC)" ;
+		fates_prt_nitr_stoich_p1:long_name = "nitrogen stoichiometry, parameter 1" ;
+
+	float fates_prt_nitr_stoich_p2(fates_prt_organs,fates_pft) ;
+		fates_prt_nitr_stoich_p2:units = "(gN/gC)" ;
+		fates_prt_nitr_stoich_p2:long_name = "nitrogen stoichiometry, parameter 2" ;
+
+	float fates_prt_phos_stoich_p1(fates_prt_organs,fates_pft) ;
+		fates_prt_phos_stoich_p1:units = "(gP/gC)" ;
+		fates_prt_phos_stoich_p1:long_name = "phosphorous stoichiometry, parameter 1" ;
+
+	float fates_prt_phos_stoich_p2(fates_prt_organs,fates_pft) ;
+		fates_prt_phos_stoich_p2:units = "(gP/gC)" ;
+		fates_prt_phos_stoich_p2:long_name = "phosphorous stoichiometry, parameter 2" ;
+
+	float fates_prt_alloc_priority(fates_prt_organs,fates_pft) ;
+		fates_prt_alloc_priority:units = "index (0-fates_prt_organs)" ;
+		fates_prt_alloc_priority:long_name = "Priority order for allocation" ;
+
+	float fates_phenflush_fraction(fates_pft) ;
+		fates_phenflush_fraction:units = "fraction" ;
+		fates_phenflush_fraction:long_name = "Upon bud-burst, the maximum fraction of storage carbon used for flushing leaves" ;
+
+	float fates_turnover_retrans_mode(fates_pft) ;
+		fates_turnover_retrans_mode:units = "index" ;
+		fates_turnover_retrans_mode:long_name = "retranslocation method for leaf/fineroot turnover" ;
+
+	float fates_turnover_carb_retrans_p1(fates_prt_organs,fates_pft) ;
+		fates_turnover_carb_retrans_p1:units = "na" ;
+		fates_turnover_carb_retrans_p1:long_name = "retranslocation of carbon in turnover, parameter 1" ;
```

Comment:
'parameter 1' seems redundant here and in the two below?

Reply:
I can remove it.  I suppose we should remove the "p1" part.   We can add the p1 back at a later date if we need to add more retranslocation parameters.